packages/suite/src/services/suite/metadata/GoogleProvider.ts

```diff
@@ -16,7 +16,7 @@ class GoogleProvider extends AbstractMetadataProvider {
             this.connected = true;
             return true;
         } catch (error) {
-            return false;
+            return error.message as string;
```

Comment:
```suggestion
            return error.toString();
```
what if message is not a string or there is no message?

Reply:
resolved with instanceof Error check